### PR TITLE
Update the dotnet vstest doc

### PR DESCRIPTION
**Pull request by singhsarab**
Removed the reference to coded ui.
Fixed the message for parallel.
Removed Framework35 from the supported frameworks.

### CODE REVIEW

**Comment by singhsarab** [comment]
@PBoraMSFT

**Comment by PBoraMSFT** [comment]
Looks good to me, @singhsarab

File: docs/core/tools/dotnet-vstest.md [hunk 1]
```diff
@@ -67,11 +67,11 @@ Target platform architecture used for test execution. Valid values are `x86`, `x
 
 `--Framework|/Framework:<Framework Version>`
 
-Target .NET Framework version used for test execution. Examples of valid values are `.NETFramework,Version=v4.6` or `.NETCoreApp,Version=v1.0`. Other supported values are `Framework35`, `Framework40`, `Framework45`, `FrameworkCore10`, and `FrameworkUap10`.
+Target .NET Framework version used for test execution. Examples of valid values are `.NETFramework,Version=v4.6` or `.NETCoreApp,Version=v1.0`. Other supported values are `Framework40`, `Framework45`, `FrameworkCore10`, and `FrameworkUap10`.
 
 `--Parallel|/Parallel`
 
-Execute tests in parallel. By default, all available cores on the machine are available for use. Set an explicit number of cores with a settings file.
+Execute tests in parallel. By default, all available cores on the machine are available for use. Specify explicit number of cores by setting MaxCpuCount property under the RunConfiguration node in the runsettings file.
```
Comment (rpetrusha) [inline docs/core/tools/dotnet-vstest.md]:
```suggestion
Execute tests in parallel. By default, all available cores on the machine are available for use. Specify an explicit number of cores by setting the MaxCpuCount property under the RunConfiguration node in the runsettings file.
```

File: docs/core/tools/dotnet-vstest.md [hunk 2]
```diff
@@ -156,11 +156,11 @@ Target platform architecture used for test execution. Valid values are `x86`, `x
 
 `--Framework|/Framework:<Framework Version>`
 
-Target .NET Framework version used for test execution. Examples of valid values are `.NETFramework,Version=v4.6` or `.NETCoreApp,Version=v1.0`. Other supported values are `Framework35`, `Framework40`, `Framework45`, and `FrameworkCore10`.
+Target .NET Framework version used for test execution. Examples of valid values are `.NETFramework,Version=v4.6` or `.NETCoreApp,Version=v1.0`. Other supported values are `Framework40`, `Framework45`, and `FrameworkCore10`.
 
 `--Parallel|/Parallel`
 
-Execute tests in parallel. By default, all available cores on the machine are available for use. Set an explicit number of cores with a settings file.
+Execute tests in parallel. By default, all available cores on the machine are available for use. Specify explicit number of cores by setting MaxCpuCount property under the RunConfiguration node in the runsettings file.
```
Comment (rpetrusha) [inline docs/core/tools/dotnet-vstest.md]:
```suggestion
Execute tests in parallel. By default, all available cores on the machine are available for use. Specify an explicit number of cores by setting the MaxCpuCount property under the RunConfiguration node in the runsettings file.
```

File: docs/core/tools/dotnet-vstest.md [hunk 3]
```diff
@@ -232,11 +232,11 @@ Target platform architecture used for test execution. Valid values are `x86`, `x
...
```
Comment (rpetrusha) [inline docs/core/tools/dotnet-vstest.md]:
```suggestion
Execute tests in parallel. By default, all available cores on the machine are available for use. Specify an explicit number of cores by setting the MaxCpuCount property under the RunConfiguration node in the runsettings file.
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks, @singhsarab. I've suggested some minor changes. You can submit them as a batch by reviewing them from the Files tab.

**Comment by singhsarab** [comment]
@rpetrusha Can you please review and approve. Thanks.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @singhsarab. I'll merge your PR now. The changes should be live on docs.microsoft.com in the next day or two.

**Comment by singhsarab** [comment]
@rpetrusha Thanks Ron.

**Pull request merged**